### ISSUE TITLE
feat(accessibility): Ensure links have sufficient contrast

### ISSUE THREAD

**Issue by breity** [WISE-Community/WISE-Client#1392]
## Description
Our `<a>` links [fail WCAG AA standards](https://webaim.org/resources/linkcontrastchecker/?fcolor=000000&bcolor=FFFFFF&lcolor=1C8DA8) due to insufficient contrast between link color and background. We should change link color and consider adding `text-decoration: underline` style to links to make them accessible.
